src/pages/registration/contributors_tab/components/AffiliationsCell.tsx

```diff
@@ -91,9 +91,13 @@ const AffiliationsCell: FC<AffiliationsCellProps> = ({ affiliations, baseFieldNa
 
   return (
     <>
-      {affiliations?.map((affiliation) => (
-        <StyledCard key={affiliation.id}>
-          <AffiliationHierarchy unitUri={affiliation.id} />
+      {affiliations?.map((affiliation, index) => (
+        <StyledCard key={affiliation.id ?? index}>
+          {affiliation.id ? (
+            <AffiliationHierarchy unitUri={affiliation.id} />
+          ) : (
+            affiliation.labels && <Typography>"{getAffiliationLabel(affiliation.labels)}"</Typography>
```

Comment:
det skal ikke stå `Forslag` ellernoe før vi viser affiliationforslaget?

Reply:
kommer knapp med "Verifiser tilknytning" etterpå, så det blir likt som for forfatter